presto-orc/src/test/java/com/facebook/presto/orc/TestDecryption.java

```diff
@@ -458,6 +467,65 @@ public void testEncryptionMultipleColumns()
                 outputColumns);
     }
 
+    @Test
+    public void testSkipFirstStripe()
+            throws Exception
+    {
+        OrcDataSource orcDataSource = new FileOrcDataSource(
+                new File(getResource("encrypted_2splits.dwrf").getFile()),
+                new DataSize(1, MEGABYTE),
+                new DataSize(1, MEGABYTE),
+                new DataSize(1, MEGABYTE),
+                true);
+        Slice iek1 = Slices.utf8Slice("key");
```

Comment:
What does `iek1` mean here?

Reply:
intermediate encryption key.  i'll just inline this, though.